### PR TITLE
Use biome

### PR DESCRIPTION
**Pull request by fernandoabolafio**

### CODE REVIEW

File: biome.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+        "noForEach": "off"
+      },
+      "performance": {
+        "noAccumulatingSpread": "off"
+      },
+      "suspicious": {
+        "noConsoleLog": "error"
+      },
+      "correctness": {
+        "noUnusedVariables": "error"
```
Comment (s-petey) [inline biome.json]: We are missing: `"noUnusedImports": "error"`
From the root config in the app.

File: biome.json [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+      },
+      "style": {
```
Comment (s-petey) [inline biome.json:52]: I think we should keep this config closer to the same as it is in the app, removing the other values other than `"useEnumInitializers": "off"`

File: biome.json [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    "lineEnding": "lf",
+    "lineWidth": 80,
+    "attributePosition": "auto"
+  },
+  "organizeImports": {
+    "enabled": true
+  },
+  "javascript": {
+    "formatter": {
+      "quoteStyle": "single"
```
Comment (s-petey) [inline biome.json]: Should we carry over more values here form the app config?

**Review by s-petey — APPROVED** [review]

**Comment by s-petey** [comment]
I've approved, however I helped with this PR, may want another once over.

**Review by fernandoabolafio — COMMENTED** [review]
Looks good to me

**Pull request merged**